packages/office-addin-telemetry/src/cli.ts

```diff
@@ -0,0 +1,22 @@
+import * as commander from "commander";
+import * as commands from "./command";
+commander.name("office-addin-telemetry");
+commander.version(process.env.npm_package_version || "(version not available)");
+commander
+    .version("0.0.1");
+commander
```

Comment:
As discussed please implement start and stop commands that take a group-name param to update the json config file